client/new-stack-editor/lib/index.coffee

```diff
@@ -36,27 +43,25 @@ module.exports = class StackEditorAppController extends AppController
     @stackEditor.on 'InitializeRequested', @bound 'initializeStack'
 
 
-  openEditor: (templateId) ->
+  openEditor: (templateId, reset = no) ->
```

Comment:
I usually not prefer that if there is only one argument, if there will be more than I can update.